frontend/src/components/ProjectColumnComponent.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div
+    class="px-2 py-2 my-2 d-flex justify-content-between align-items-center"
+    :class="{ active: isActive }"
+    @click.prevent="setupProject"
+    @dblclick.prevent="enableEditMode"
+    @mouseover="isHover = true"
+    @mouseleave="isHover = false"
+  >
+    <div class="d-flex align-items-center">
+      <div v-if="!isEditMode" class="text-light fs-4 pe-1 me-1">
+        <font-awesome-icon :icon="['fa', 'file']" />
+      </div>
+      <input
+        ref="inputElement"
+        v-else
+        type="text"
+        class="form-control"
+        v-model="projectName"
+        @blur="disableEditMode"
+      />
+      <div v-if="!isEditMode" class="fw-semibold text-light">
+        {{ projectName }}
+      </div>
+    </div>
+    <div v-if="!isHover" class="expire text-white-50">
+      {{ leftDays }}
+    </div>
+    <button v-else type="button" class="btn text-light" @click.stop="deleteProject">
+      <font-awesome-icon :icon="['fas', 'trash']" />
+    </button>
+  </div>
+</template>
+
+<script setup lang="ts">
+import type { Project } from '@/constants'
+import { useAppStore, useProjectIdStore } from '@/stores/stores'
+import moment from 'moment'
+import { computed, ref, watch, type Ref } from 'vue'
+
+const props = defineProps<{ project: Project }>()
+const emit = defineEmits<{ deleteProject: [Project] }>()
+
+const leftDays = computed(() =>
+  props.project.expire_on ? `Expire ${moment.utc(props.project.expire_on).fromNow()}` : ''
```

Comment:
Is there any way this gets updated? This is set on the backend upon validating the first file. We could maybe set it in JS upon upload completion of the first file. Returning user would retrieve it from the API when calling `/project`.

Reply:
OK, can you open a ticket to ass this to the API?